torch_geometric/graphgym/utils/agg_runs.py

```diff
@@ -86,8 +86,8 @@ def agg_runs(dir, metric_best='auto'):
         validation performance. Options: auto, accuracy, auc.
 
     '''
-    results = {'train': None, 'val': None, 'test': None}
-    results_best = {'train': None, 'val': None, 'test': None}
+    results = {'train': None, 'val': None}
```

Comment:
test loop will not run during the training.

Reply:
Thanks!